diff --git a/dist/forms-ByF1AEEn.js b/dist/forms-ByF1AEEn.js
index a727d6b998eeb9126d25ab14c85dcab35c44408d..5a0716b2900693419ddd2ca1b7c2e6b64fbdb0dd 100644
--- a/dist/forms-ByF1AEEn.js
+++ b/dist/forms-ByF1AEEn.js
@@ -45,3 +45,3 @@
-var PutioApiError = class extends Schema.TaggedErrorClass()("PutioApiError", { ...PutioResponseErrorFields }) {};
-var PutioAuthError = class extends Schema.TaggedErrorClass()("PutioAuthError", { ...PutioResponseErrorFields }) {};
-var PutioRateLimitError = class extends Schema.TaggedErrorClass()("PutioRateLimitError", {
+var PutioApiError = class extends Schema.TaggedError()("PutioApiError", { ...PutioResponseErrorFields }) {};
+var PutioAuthError = class extends Schema.TaggedError()("PutioAuthError", { ...PutioResponseErrorFields }) {};
+var PutioRateLimitError = class extends Schema.TaggedError()("PutioRateLimitError", {
